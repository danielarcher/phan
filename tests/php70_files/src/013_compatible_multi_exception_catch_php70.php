<?php
class FirstException extends \Exception
{

}

class SecondException extends \Exception
{

}

try {

} catch (FirstException | SecondException $e) {
    
}
