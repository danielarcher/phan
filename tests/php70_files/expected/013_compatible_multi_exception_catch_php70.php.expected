%s:14 PhanCompatibleMultiExceptionCatchPHP70 Catching multiple exceptions is not supported before PHP 7.1
